css/main.css

```diff
@@ -0,0 +1,165 @@
+.article-header-culture a:hover { color: #000; text-decoration: none; }
+.block-caption { position: absolute; bottom: 0; color: #df0000; background-color: #fbfbfb; font-family: 'Roboto Condensed', sans-serif; font-weight: bold; text-transform: uppercase; pointer-events: none; }
+.article-tag { background-color: #d6f5f5; padding: 5px; }
+.article-tag:hover { background-color: #000; color: #fff; }
+.article-tag a:hover { color: #fff; }
+.article-box { font-family: 'Roboto Condensed', sans-serif; font-weight: bold; line-height: 1.25em; }
+.article-box div { margin-bottom: 0.75rem; }
+.article-box div:last-child { margin-bottom: 0; }
+.article-box a { font-size: 1.25rem; }
+
+/* Slideshow Section */
+.second-N { position: absolute; bottom: 6px; right: 14px; width: 36px; height: 36px; }
+
+.slide-caption {
+  position: absolute;
+  width: 68px;
+  height: 52px;
+  background-color: #fbfbfb;
+  bottom: 0;
+  font-family: 'Roboto Condensed', sans-serif;
+  font-weight: bold;
+  font-size: 1.25rem;
+  pointer-events: none;
+}
+.slide-caption-box { position: absolute; bottom: 0; right: 8px; width: 42px; height: 36px; border: 2px #df0000 solid; }
+.slide-caption-box-2 { position: absolute; bottom: 5px; right: 13px; width: 42px; height: 36px; border: 2px #df0000 solid; border-bottom: 0; border-right: 0; }
+.slide-caption-box-3 { position: absolute; bottom: 10px; right: 18px; width: 42px; height: 36px; border: 2px #df0000 solid; border-bottom: 0; border-right: 0; }
+
+/* Footer Subscribe and Newsletter Section */
+.book-case { padding: 122px 150px; position: relative; }
+.book-shelf { position: absolute; margin-bottom: 1rem; top: 0; left: 0; }
+.book-behind { position: absolute; z-index: 1; }
+.book-forefront { position: absolute; left: 90px; top: 13px; z-index: 2; }
+
+.subscribe-bg { background: url(../images/Footer/subscribe-img2.gif) center center / cover no-repeat; width: 48%; min-height: 160px; }
+
+.social-media { font-size: 28px; }
+.social-media a { margin-left: 0.65rem; color: gray; }
+.social-media a:first-of-type { margin-left: 0; }
+
+/* facebook icon */
+.social-media a:nth-child(1):hover { color: midnightblue; }
+
+/* twitter icon */
+.social-media a:nth-child(2):hover { color: skyblue; }
+
+/* instagram icon */
+.social-media a:nth-child(3):hover { color: blue; }
+
+/* tumbler icon */
+.social-media a:nth-child(4):hover { color: darkblue; }
+
+/* linkedIn icon */
+.social-media a:nth-child(5):hover { color: dodgerblue; }
+
+.footer-links { list-style: none; font-size: 0.9rem; font-weight: 500; color: #727272; }
+.footer-links li { border-right: 1px gray solid; padding: 0 0.85rem; line-height: 1rem; margin: 0.35rem 0; }
+.footer-links li:last-of-type { border: none; }
+.footer-links li a { color: #727272; }
+.footer-links li a:hover { color: #111; }
+
+/* Horizontal Line Dividers */
+.gray-divider { border-bottom: 2px #f2f2f2 solid; width: 96%; height: 2px; margin: auto; }
```

Comment:
Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

`mx-auto` Bootstrap class name can be used instead. See [horizontal centering](https://getbootstrap.com/docs/4.1/utilities/spacing/#horizontal-centering) in bootstrap documentation.

#### Please, lets do this for all other occurrences. :blush:

Reply:
I know of mx-auto but I was creating a custom class anyway (since bootstrap only allows for 1px borders and isn't sufficient enough for what I wanted) so I figured I might as well just put all of needs of the class within that one custom class one time...

Reply:
I understand.
I suggested this because the objective of this project is to use bootstrap **everywhere** possible.